src/App.js

```diff
@@ -1,5 +1,5 @@
 import './App.css';
-import NavBar from './Components/NavBar/NavBar';
+import NavBar from './Components/NavBar/NavBar.jsx';
```

Comment:
Why some files with extension and others not? This is inconsistent, all with extension, or no one with extensions

Reply:
Done